test/integration/testLargeResultSet.js

```diff
@@ -113,4 +113,126 @@ describe('Large result Set Tests', function ()
+
+    var arrJSON = [];
+    const sampleJSON = {
+      "root":
+      {
+        "key":
+          [
+            {
+              "key1": "value1",
+              "key2": "value2",
+              "key3": "value3",
+              "key4": "value4",
+              "key5":
+              {
+                "key":
+                  [
+                    { "key1": "value1", "key2": "value2" },
+                    { "key1": "value1", "key2": "value2" },
+                    { "key1": "value1", "key2": "value2" },
+                    { "key1": "value1", "key2": "value2" }
+                  ]
+              },
+              "key6":
+                [
+                  { "key1": "value1", "key": "value" },
+                  { "key1": "value1", "key": "value" },
+                  { "key1": "value1", "key": "value" },
+                  { "key1": "value1", "key": "value" },
+                  { "key1": "value1", "key": "value" },
+                  { "key1": "value1", "key": "value" },
+                  { "key1": "value1", "key": "value" }
+                ]
+            },
+          ]
+      }
+    };
+
+    for (var i = 0; i < sourceRowCount; i++)
+    {
+      arrJSON.push(sampleJSON);
+    }
+
+    async.series([
+      function (callback)
+      {
+        testUtil.executeCmd(connection, createTableWithVariant, callback);
+      },
+      function (callback)
+      {
+        connection.execute({
+          sqlText: insertVariant,
+          binds: [JSON.stringify(arrJSON)],
+          complete: function (err, stmt)
+          {
+            if (err)
+            {
+              callback(err);
+            }
+
+            try
+            {
+              assert.strictEqual(stmt.getNumUpdatedRows(), sourceRowCount);
+              callback();
+            }
+            catch (err)
+            {
+              callback(err);
+            }
+          }
+        });
+      },
+      function (callback)
+      {
+        connection.execute({
+          sqlText: selectVariant,
+          streamResult: true,
+          complete: function (err, stmt)
+          {
+            if (err)
+            {
+              callback(err);
+            }
+
+            var stream = stmt.streamRows();
+            var rowCount = 0;
+            stream.on('data', function ()
+            {
+              rowCount++;
+            });
+            stream.on('error', function (err)
+            {
+              callback(err);
+            });
+            stream.on('end', function ()
+            {
+              try
+              {
+                assert.strictEqual(rowCount, sourceRowCount);
+                callback();
+              }
+              catch (err)
+              {
+                callback(err);
+              }
+            });
+          }
+        });
+      },
+      function (callback)
+      {
+        testUtil.executeCmd(connection, dropTableWithVariant, callback);
```

Comment:
the main focus of the test is to insert and select from table with variant - maybe we can wrap this test in describe and create table put in `before` and drop table in `after`?

Reply:
Ok, placed the test in its own describe and create/drop table are now in the before/after of the test